frontend/src/metabase/admin/datamodel/containers/MetricForm.jsx

```diff
@@ -52,7 +51,7 @@ import Table from "metabase-lib/lib/metadata/Table";
       return errors;
     },
   },
-  (state, props) => metricFormSelectors(state, props),
+  (state, props) => ({ ...props, initialValues: props.metric }),
```

Comment:
I don't think you need to merge `props` here (it's basically like `mapStateToProps` which adds additional props)